blocks/graph.py

```diff
@@ -125,9 +125,11 @@ def _get_variables(self):
                                  for scan in self.scans]
 
             seen = set()
-            main_vars = [var for var in list(chain(
-                *[apply_node.inputs for apply_node in sorted_apply_nodes]))
-                if not (var in seen or seen.add(var))] + self.outputs
+            main_vars = (
+                [var for var in list(chain(
+                    *[apply_node.inputs for apply_node in sorted_apply_nodes]))
+                 if not (var in seen or seen.add(var))]
+                + [var for var in self.outputs if var not in seen])
```

Comment:
I don't understand why Scrutinizer doesn't complain, but it does on my computer: PEP8 says that binary operators should go before the line break i.e.
```python
if not ...] +
[var for var ...
```